src/main/frontend/components/block.cljs

```diff
@@ -1787,9 +1788,77 @@
                   (= move-to :nested)))
           (dnd-separator move-to block-content?))))))
 
+(rum/defcs decrypt-block  < rum/reactive
+  {:init (fn [state]
+           (state/update-state! :editor/content {})
+           state
+           )
```

Comment:
The closing parentheses and the last line of the function can be in the same line. Please refer to this great [clojure style guide](https://github.com/bbatsov/clojure-style-guide).

Reply:
BTW, your Clojurescript code is great!

Reply:
> 

Wow, this means a lot to me!